models/git/lfs.go

```diff
@@ -135,25 +135,18 @@ var ErrLFSObjectNotExist = db.ErrNotExist{Resource: "LFS Meta object"}
 // NewLFSMetaObject stores a given populated LFSMetaObject structure in the database
 // if it is not already present.
 func NewLFSMetaObject(ctx context.Context, repoID int64, p lfs.Pointer) (*LFSMetaObject, error) {
-	ctx, committer, err := db.TxContext(ctx)
```

Comment:
Are we here inside a double transaction?
Or why was this removed without replacement?

Reply:
This operation only involves a single INSERT statement, so wrapping it in a transaction is unnecessary. The database guarantees atomicity for individual SQL statements by default.